packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, { StorybookEmailArgs } from '../../storybook-email';
+// const configURL = require('../../../../../config').get('contentServer.url')
```

Comment:
@jody When I'm importing the config file in Storybook, I'm running into errors because that config file imports modules like `fs` which cannot be rendered in browser so keeping it like this for now. Will get back to this tomorrow!

Reply:
I think you'll want to require this in `mjml-server` and then send it down to the templates as a new variable.

BTW, I think this file can also be a `.ts` file rather than a `.tsx` file.